terraform/environments/staging/main.tf

```diff
@@ -536,6 +536,136 @@ resource "google_project_iam_member" "application" {
   member = "serviceAccount:${each.value}"
 }
 
+# Deploy relays
+
+module "relays" {
+  source     = "../../modules/relay-app"
+  project_id = module.google-cloud-project.project.project_id
+
+  instances = {
+    "asia-east1" = {
+      "asia-east1-a" = {
+        type     = "n1-standard-1"
+        replicas = 1
+      }
+    }
+
+    "asia-south1" = {
+      "asia-south1-a" = {
+        type     = "n1-standard-1"
+        replicas = 1
+      }
+    }
+
+    "australia-southeast1" = {
+      "australia-southeast1-a" = {
+        type     = "n1-standard-1"
+        replicas = 1
+      }
+    }
+
+    "me-central1" = {
+      "me-central1-a" = {
+        type     = "n1-standard-1"
+        replicas = 1
+      }
+    }
+
+    "europe-west1" = {
+      "europe-west1-d" = {
+        type     = "n1-standard-1"
+        replicas = 1
+      }
+    }
+
+    "southamerica-east1" = {
+      "southamerica-east1-b" = {
+        type     = "n1-standard-1"
+        replicas = 1
+      }
+    }
+
+    "us-east1" = {
+      "us-east1-d" = {
+        type     = "n1-standard-1"
+        replicas = 1
+      }
+    }
+
+    "us-west2" = {
+      "us-west2-b" = {
+        type     = "n1-standard-1"
+        replicas = 1
+      }
+    }
+
+    "us-central1" = {
+      "us-central1-b" = {
+        type     = "n1-standard-1"
+        replicas = 1
+      }
+    }
+
+  }
+
+  vpc_network = "projects/${var.project_id}/global/networks/default"
+
+  container_registry = module.google-artifact-registry.url
+
+  image_repo = module.google-artifact-registry.repo
+  image      = "relay"
+  image_tag  = var.relay_image_tag
+
+  observability_log_level = "debug"
+
+  application_name    = "relay"
+  application_version = "0-0-1"
+
+  application_ports = [
+    {
+      name     = "TURN"
+      protocol = "UDP"
+      port     = 3478
```

Comment:
We _won't_ be listening on this port until the portal sends us the init message. We should take this into account in whatever health-check we end up implementing.